stories/CssVariables.stories.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { CssVariables } from '../src'
+
+const App = ({ children }) => <div>{children}</div>
+
+storiesOf('CssVariables', module).add('Default', () => (
+    <App>
+        <CssVariables />
+
+        <p>
```

Comment:
I think this text should be in the jsdoc block of the component.
Rather than explaining here what you could do with the `CssVariables` component,
I'd be expecting a show-case, maybe something like this:

```jsx
<p>
  This {' '}
  <span style="color: var(--colors-blue500);">text</span> {' '}
  contains snippets with different {' '}
  <span style="color: var(--colors-red500);">colors</span>
</p>
```

Reply:
Yeah I just based it on the text of the CssReset component. I'll modify it to demonstrate what it does directly 👍

Reply:
We could make this text more specific, let me know what you think. Alongside this we could also add a story which doesn't rely on the default value for `variables` but sets its own. Just to demonstrate how that'd work. What do you think?

Reply:
Yeah, we should have one story for every possible scenario

Reply:
I resolved this because CssVariables and ThemeCssVariables both have a story. Maybe we need to update the wording, but to me it seems all right.